src/main.js

```diff
@@ -102,7 +102,16 @@ var load = loader({
   server: {
     requires: ['cfg', 'router', 'docs'],
     setup: ({cfg, router, docs}) => {
-      let hooksApp = app({docs,cfg.server});
+      let hooksApp = app({
+        port:                   cfg.server.port,
+        env:                    cfg.server.env,
+        forceSSL:               cfg.server.forceSSL,
+        trustProxy:             cfg.server.trustProxy,
+        contentSecurityPolicy:  cfg.server.contentSecurityPolicy,
+        robotstxt:              cfg.server.robotstxt,
+        rootDocsRedirect:       true, 
+        docs,
+      });
```

Comment:
this part looks fine

Reply:
So this can be merged only after we fix tc-lib-app , right? PR in tc-lib-docs was merged.

Reply:
Yes, that's correct.